docker-hub/billing/faq.md

```diff
@@ -93,26 +83,26 @@ When you add seats to your plan in the middle of your billing cycle, a prorated
 
 ### How do I upgrade to a Pro plan from a legacy individual plan?
 
-Upgrading your legacy plan to a Pro plan offers you unlimited public and private repos and 1 collaborator or service account. For information on how to upgrade to a Pro plan from a legacy plan, see the [How To Upgrade](https://docs.docker.com/docker-hub/billing/upgrade/y){:target="blank" rel="noopener" class=""} page.
+Upgrading your legacy plan to a Pro plan offers you unlimited public and private repos and 1 collaborator or service account. For information on how to upgrade to a Pro plan from a legacy plan, see the [How To Upgrade](upgrade.md){:target="blank" rel="noopener" class=""} page.
 
 ### How do I upgrade to a Team plan from a legacy organization plan?
 
-Upgrading your legacy plan to a Team plan offers you unlimited private repos, unlimited teams, and 3 parallel builds. For information on how to upgrade to a Team plan from a legacy (per-repository) plan, see the [How To Upgrade](https://docs.docker.com/docker-hub/billing/upgrade/){:target="blank" rel="noopener" class=""} page.
+Upgrading your legacy plan to a Team plan offers you unlimited private repos, unlimited teams, and 3 parallel builds. For information on how to upgrade to a Team plan from a legacy (per-repository) plan, see the [How To Upgrade](upgrade.md){:target="blank" rel="noopener" class=""} page.
 
 ### How do downgrades from a Pro or Team plan work?
 When you downgrade your Pro or Team plan, changes are applied at the end of your billing cycle. For example, if you are currently on a Team plan which is billed on the 8th of every month and you choose to downgrade to a Free Team plan on the 15th, your Team plan will be active until 7th of the following month. You will be transferred to a Free Team plan on the 8th of the following month. Please refer to the next two questions below for instructions on how to downgrade your account.
 
 ### How do I downgrade from a Team plan to a Free Team plan?
 
-Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, [see the billing](https://docs.docker.com/docker-hub/billing/){:target="blank" rel="noopener" class=""} page.
+Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, see [the billing](downgrade.md){:target="blank" rel="noopener" class=""} page.
```

Comment:
@jerae-duffin Not sure this is a typo, but I believe we should be pointing users to the Pricing page for info on what's included in the Free plan. This section points to the Downgrade page instead.